platformio.ini

```diff
@@ -401,8 +401,8 @@ lib_deps = ${esp32c3.lib_deps}
 [env:esp32s3dev_8MB]
 ;; ESP32-S3-DevKitC-1 development board, with 8MB FLASH, no PSRAM
 board = esp32-s3-devkitc-1
-platform = espressif32@5.1.1
-platform_packages = platformio/framework-arduinoespressif32@3.20004.220825
+platform = espressif32@5.2.0
```

Comment:
Se my comment. Can we stay with espressif32@5.1.1? What are the compile errors you experienced?